docs/modules/manual-testing/pages/device-controls.adoc

```diff
@@ -1,158 +1,151 @@
-= Device controls
-:navtitle: Device controls
-
-Use these device controls to interact with your device during a manual test session.
-
-== Touch
-
-To interact with your device, select *Touch*, then use *left-click* to tap and swipe the device screen.
-
-image:manual-testing:select-touch-context.png[width=1000,alt="Select Touch on Manual session"]
-
-== Pinch/zoom
-
-To pinch or use two-finger zoom, select *Pinch/ Zoom,* then place the center circle on the target and drag an outer circle.
-
-image:manual-testing:device-controls-context3.png[width=1000,alt="A manual session with a closeup of a device using pinch/zoom."]
-
-== Set device location
-
-To change the device location, select *Location*, then search for a location or enter the longitude and latitude.
-
-image:manual-testing:device-controls-closeup4.png[width=400,alt="A manual session with a closeup of the location controls."]
-
-== Set device timezone
-
-To change the device timezone, select *Set device timezone* and choose a timezone from the dropdown.
-
-image:manual-testing:device-controls-closeup3.png[width=300,alt="A manual session with a closeup of a device set to landscape."]
-
-== Rotate screen
-
-To rotate the device screen between portrait or landscape, select *Rotate Screen*.
-
-image:manual-testing:device-controls-context2.png[width=1000,alt="A manual session with a rotate screen"]
-
-== Volume
-
-To change the device volume, select *Volume Up* or *Volume Down*.
-
-image:manual-testing:volume-up-volume-down-context.png[width=1000,alt="Select Volume up or Volume Down"]
-
-[#_adb_shell]
-== `adb` shell
-
-To enter Android Debug Bridge (`adb`) commands, select *adb shell*, then enter a xref:manual-testing:adb-commands.adoc[supported `adb` command].
-
-image:manual-testing:adb-shell-closeup.png[width="750",alt="A manual session with a closeup of the adb shell running `adb shell dumpsys display`."]
-
-[#_perform_voiceover]
-== Perform VoiceOver
-
-Once xref:devices:local-devices/configure-ios-voiceover.adoc[VoiceOver for iOS] is set up, select *Perform VoiceOver*, then choose a VoiceOver option to test an app's screen reader compatibility.
-
-image:manual-testing:perform-voiceover-context.png[width=1000,alt="Select Perform Voiceover feature"]
-
```

Comment:
This was the section, I removed, I'm not really why the GitHub is showing the entire page as edited